compiler/passes/docs.cpp

```diff
@@ -84,7 +84,7 @@ void docs(void) {
     mkdir(folderName.c_str(), S_IWUSR|S_IRUSR|S_IXUSR);
     
     forv_Vec(ModuleSymbol, mod, gModuleSymbols) {
-      if (!devOnlyModule(mod) || developer) {
+      if ((!devOnlyModule(mod) || developer) && !mod->hasFlag(FLAG_NO_DOC)) {
```

Comment:
I don't think it really matters, but it seems like putting the `!mod->hasFlag(FLAG_NO_DOC)` check first would take better advantage of boolean short circuiting.